tests/Makefile

```diff
@@ -3,7 +3,7 @@
 # To compile with PKCS11: add "-lpkcs11-helper" to LDFLAGS
 
 CFLAGS	?= -O2
-WARNING_CFLAGS ?= -Wall -W -Wdeclaration-after-statement -Wno-unused-function -Wno-unused-value
+WARNING_CFLAGS ?= -Wall -W -Wdeclaration-after-statement -Wunused-function -Wunused-value
```

Comment:
Consider using `-Wunused` (which is -Wunused-function -Wunused-label -Wunused-parameter -Wunused-value -Wunused-variable), if we can.

Reply:
Remove `-Wunused-function -Wunused-value`. They're already included in `-Wall`.